unsafe/src/main/java/org/apache/spark/unsafe/bitset/BitSet.java

```diff
@@ -102,4 +102,12 @@ public boolean isSet(int index) {
   public int nextSetBit(int fromIndex) {
     return BitSetMethods.nextSetBit(baseObject, baseOffset, fromIndex, numWords);
   }
+
+  /*
+   * @return whether any bit in the BitSet is set
+   */
+  public boolean anySet() {
+    return BitSetMethods.anySet(baseObject, baseOffset, numWords*BitSetMethods.WORD_SIZE);
```

Comment:
As I mentioned in an earlier comment, we can assume that `BitSetMethods.anySet` will be called with a size that's word aligned, so for consistency with `nextSetBit` I think we should change `anySet` to accept a size that's measured in numbers of words.  This would let us avoid the mod calculations or multiplications by word size.